pkg/podcgroup/podcgroup.go

```diff
@@ -53,7 +53,7 @@ func podContainerIDExtractor(cgroupPath string) (podUID, containerID string) {
 
 	podUID = podIDMatches[1]
 	podUID = strings.Replace(podUID, "_", "-", -1)
-	containerID = containerRuntime + "://" + containerIDMatches[2]
+	containerID = containerIDMatches[2]
```

Comment:
Shouldn't line 50 be completed to test for containerd too?
```
if containerRuntime != "docker" && containerRuntime != "cri-o" && containerRuntime != "containerd" {
```

Reply:
`containerd` does add any prefix to the cgroups path. I changed this logic a bit to consider then the prefix is `""` without assuming it's gonna be `docker`.

Reply:
Ah, good to know